File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/FormatsItem.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class FormatsItem{

	@SerializedName("itag")
	private int itag;

	@SerializedName("fps")
	private int fps;

	@SerializedName("projectionType")
	private String projectionType;

	@SerializedName("bitrate")
	private int bitrate;

	@SerializedName("mimeType")
	private String mimeType;

	@SerializedName("audioQuality")
	private String audioQuality;

	@SerializedName("approxDurationMs")
	private String approxDurationMs;

	@SerializedName("url")
	private String url;

	@SerializedName("audioSampleRate")
	private String audioSampleRate;

	@SerializedName("quality")
	private String quality;

	@SerializedName("qualityLabel")
	private String qualityLabel;

	@SerializedName("audioChannels")
	private int audioChannels;

	@SerializedName("width")
	private int width;

	@SerializedName("lastModified")
	private String lastModified;

	@SerializedName("height")
	private int height;

	@SerializedName("contentLength")
	private String contentLength;

	@SerializedName("averageBitrate")
	private int averageBitrate;

	public void setItag(int itag){
		this.itag = itag;
	}

	public int getItag(){
		return itag;
	}

	public void setFps(int fps){
		this.fps = fps;
	}

	public int getFps(){
		return fps;
	}

	public void setProjectionType(String projectionType){
		this.projectionType = projectionType;
	}

	public String getProjectionType(){
		return projectionType;
	}

	public void setBitrate(int bitrate){
		this.bitrate = bitrate;
	}

	public int getBitrate(){
		return bitrate;
	}

	public void setMimeType(String mimeType){
		this.mimeType = mimeType;
	}

	public String getMimeType(){
		return mimeType;
	}

	public void setAudioQuality(String audioQuality){
		this.audioQuality = audioQuality;
	}

	public String getAudioQuality(){
		return audioQuality;
	}

	public void setApproxDurationMs(String approxDurationMs){
		this.approxDurationMs = approxDurationMs;
	}

	public String getApproxDurationMs(){
		return approxDurationMs;
	}

	public void setUrl(String url){
		this.url = url;
	}

	public String getUrl(){
		return url;
	}

	public void setAudioSampleRate(String audioSampleRate){
		this.audioSampleRate = audioSampleRate;
	}

	public String getAudioSampleRate(){
		return audioSampleRate;
	}

	public void setQuality(String quality){
		this.quality = quality;
	}

	public String getQuality(){
		return quality;
	}

	public void setQualityLabel(String qualityLabel){
		this.qualityLabel = qualityLabel;
	}

	public String getQualityLabel(){
		return qualityLabel;
	}

	public void setAudioChannels(int audioChannels){
		this.audioChannels = audioChannels;
	}

	public int getAudioChannels(){
		return audioChannels;
	}

	public void setWidth(int width){
		this.width = width;
	}

	public int getWidth(){
		return width;
	}

	public void setLastModified(String lastModified){
		this.lastModified = lastModified;
	}

	public String getLastModified(){
		return lastModified;
	}

	public void setHeight(int height){
		this.height = height;
	}

	public int getHeight(){
		return height;
	}

	public void setContentLength(String contentLength){
		this.contentLength = contentLength;
	}

	public String getContentLength(){
		return contentLength;
	}

	public void setAverageBitrate(int averageBitrate){
		this.averageBitrate = averageBitrate;
	}

	public int getAverageBitrate(){
		return averageBitrate;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/FormatsItem.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class FormatsItem : Parcelable, Serializable {
    @SerializedName("itag")
    var itag = 0

    @SerializedName("fps")
    var fps = 0

    @SerializedName("projectionType")
    var projectionType: String? = null

    @SerializedName("bitrate")
    var bitrate = 0

    @SerializedName("mimeType")
    var mimeType: String? = null

    @SerializedName("audioQuality")
    var audioQuality: String? = null

    @SerializedName("approxDurationMs")
    var approxDurationMs: String? = null

    @SerializedName("url")
    var url: String? = null

    @SerializedName("audioSampleRate")
    var audioSampleRate: String? = null

    @SerializedName("quality")
    var quality: String? = null

    @SerializedName("qualityLabel")
    var qualityLabel: String? = null

    @SerializedName("audioChannels")
    var audioChannels = 0

    @SerializedName("width")
    var width = 0

    @SerializedName("lastModified")
    var lastModified: String? = null

    @SerializedName("height")
    var height = 0

    @SerializedName("contentLength")
    var contentLength: String? = null

    @SerializedName("averageBitrate")
    var averageBitrate = 0
    override fun toString(): String {
        return "FormatsItem{" +
                "itag = '" + itag + '\'' +
                ",fps = '" + fps + '\'' +
                ",projectionType = '" + projectionType + '\'' +
                ",bitrate = '" + bitrate + '\'' +
                ",mimeType = '" + mimeType + '\'' +
                ",audioQuality = '" + audioQuality + '\'' +
                ",approxDurationMs = '" + approxDurationMs + '\'' +
                ",url = '" + url + '\'' +
                ",audioSampleRate = '" + audioSampleRate + '\'' +
                ",quality = '" + quality + '\'' +
                ",qualityLabel = '" + qualityLabel + '\'' +
                ",audioChannels = '" + audioChannels + '\'' +
                ",width = '" + width + '\'' +
                ",lastModified = '" + lastModified + '\'' +
                ",height = '" + height + '\'' +
                ",contentLength = '" + contentLength + '\'' +
                ",averageBitrate = '" + averageBitrate + '\'' +
                "}"
    }
}